packages/devextreme-angular/src/ui/tabs/index.ts

```diff
@@ -390,6 +390,19 @@ export class DxTabsComponent extends DxComponent implements OnDestroy, OnChanges
     }
 
 
+    /**
```

Comment:
Can I find info in DX WIKI `what I should add in wrappers` when I add new options to component?

Reply:
You should use npm command: npm run regenerate-all from root.